neuro_san/http_sidecar/http_sidecar.py

```diff
@@ -100,10 +100,10 @@ def __call__(self, grpc_server: AgentServer):
 
         # Wait for "go" signal which will be set by gRPC server and corresponding machinery
         # when everything is ready for servicing.
-        event_set = self.start_event.wait(timeout=self.TIMEOUT_TO_START_SECONDS)
-        if not event_set:
-            self.logger.error({}, "Timeout (%d sec) waiting for HTTP server to start", self.TIMEOUT_TO_START_SECONDS)
-            return
+        while not self.start_event.wait(timeout=self.TIMEOUT_TO_START_SECONDS):
+            self.logger.error({},
+                              "Timeout (%d sec) waiting for signal to HTTP server to start",
+                              self.TIMEOUT_TO_START_SECONDS)
```

Comment:
Wait for as long as it takes - log this process so problem will not be hidden in output dump.